boot.js

```diff
@@ -192,11 +192,7 @@ Boot.prototype._addPlugin = function (plugin, opts, isAfter) {
   }
 
   // we add the plugin to be loaded at the end of the current queue
-  current.enqueue(obj, (err) => {
-    if (err) {
-      this._error = err
-    }
-  })
+  current.enqueue(obj, (err) => { err && (this._error = err) })
```

Comment:
I believe splitting into three lines is better for git diff or future changes. 
```suggestion
  current.enqueue(obj, (err) => { 
    err && (this._error = err) 
  })
```